packages/tkg-clusterclass-aws/bundle/config/upstream/base.yaml

```diff
@@ -45,6 +45,20 @@ spec:
       name: #@ "tkg-aws-default-{}-cluster".format(data.values.AWS_CLUSTER_CLASS_VERSION)
       namespace: #@ data.values.NAMESPACE
   variables:
+  - name: etcdExtraArgs
+    required: false
+    schema:
+      openAPIV3Schema:
+        type: object
+        additionalProperties:
+          type: string
+        default: {}
```

Comment:
Is that possible to set default value as {cipher-suites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384}, so you can remove the tlsCipherSuites variable?

Reply:
Exactly. Users must specify cipher-suites in the variable etcdExtraArgs if we don't append the default value. E.g: If users only specify the following args, cipher-suites will be missed.
```
    - name: etcdExtraArgs
      value:
        election-timeout: "2000"
        heartbeat-interval: "300"
```

Reply:
variable tlsCipherSuites will be shared as the default cipher suites by all components: apiserver, kube-controller-mananger, kubelet etc. I just don't have time to change them.